### ISSUE TITLE
Incorrect line for __location.endTag when the start tag contains newlines

### ISSUE THREAD

**Issue by rictic** [inikulin/parse5#166]
This program:

```javascript
var parse5 = require('parse5');
var document = parse5.parse('<i\n></i>', {locationInfo: true});
var elementLocation =
    document.childNodes[0].childNodes[1].childNodes[0].__location;
console.log(elementLocation.startTag);
console.log(elementLocation.endTag);
```

Outputs the following with parse5 v2.2.3:

```js
{ line: 1, col: 1, startOffset: 0, endOffset: 4 }
{ line: 1, col: 2, startOffset: 4, endOffset: 8 }
```

The offsets and columns are correct, but the end tag location's line should be `2`.

**Comment by rictic** [comment]
Thanks for the fix! Do you have an idea for when the next release stamp will be? This will fix some issues people have been seeing in our editor plugins.

**Comment by inikulin** [comment]
Hi @rictic,
I'll try to roll out next release on Monday or Tuesday.

**Comment by rictic** [comment]
Awesome, thanks!

**Comment by inikulin** [comment]
@rictic just released `v3.0.0` which includes this fix
